usr.sbin/kldxref/ef.c

```diff
@@ -338,6 +344,23 @@ ef_parse_dynamic(elf_file_t ef, const GElf_Phdr *phdyn)
 				goto out;
 			}
 			break;
+		case DT_RISCV_CHERI___CAPRELOCS:
```

Comment:
Arguably this should be a separate switch that's got `if (elf_machine(...) != EM_RISCV))` around it in case these ever alias some other DT_ARCH_FOO that kldxref also needs to care about, but caprelocs will die on a day that will be celebrated, so I don't foresee it actually being a problem for us.